lib/Jit/LLILCJit.cpp

```diff
@@ -303,6 +283,16 @@ CorJitResult LLILCJit::compileMethod(ICorJitInfo *JitInfo,
 
       // Tell the CLR that we've successfully generated code for this method.
       Result = CORJIT_OK;
+
+      if (Context.Options->DumpLevel >= ::DumpLevel::SUMMARY) {
+         errs() << "LLILC compiled: " 
+                << ", Entry = " << *NativeEntry 
+                << ", End = " << (*NativeEntry + *NativeSizeOfCode)
+                << ", size = " << *NativeSizeOfCode 
+                << " method = " << Context.MethodName 
+                << '\n';
+      }
+
```

Comment:
This block is moved before the verification of the bitcode.  Is that the right thing to do?  I would have thought we wanted to run the IR check to ensure that we have a legal translation.